action.yml

```diff
@@ -48,7 +48,7 @@ inputs:
     required: false
     default: true
   registry_secrets:
-    description: Image registry credentrials (Balena secrets.json)
+    description: Image registry credentials (Balena secrets.json)
```

Comment:
I would prefer that fixing typos or whitespace be in a separate commit.

Some reasoning is that reverting, or cherry-picking, functional commits is much easier when it doesn't include unrelated changes.

It's also excellent practice for rebasing and splitting commits!